OpenSim/Simulation/Model/Model.cpp

```diff
@@ -1686,7 +1686,7 @@ void Model::writeMarkerFile(const string& aFileName)
  * @param aMarkerSet The new marker set to copy.
  * @return Number of markers that were successfully added to the model.
  */
-int Model::replaceMarkerSet(const SimTK::State& s, const MarkerSet& aMarkerSet)
+int Model::replaceMarkerSet(SimTK::State& s, const MarkerSet& aMarkerSet)
```

Comment:
> If that's not recommended inside the method, I'm happy to make the call from OpenSimContext

That's correct.

> we need to unify these two methods (update/replace) to make sure the API and GUI are executing the same workflow

I agree. PR #1783 makes them consistent in that they both now do not call initSystem().